test/e2e/fixtures.go

```diff
@@ -99,6 +99,18 @@ func skipIfHasWindowsNodes(tb testing.TB) {
 	}
 }
 
+func skipIfHasEnvSkipLongRunCases(tb testing.TB) {
+	if os.Getenv("SkipLongRunCases") == "true" {
```

Comment:
Not sure which is better but I prefer making `SkipLongRunCases`, `NotDefault` options. What's your opinion?
https://github.com/antrea-io/antrea/blob/a8db01110087feff7ce7b96cdd31b8be544ce3fb/test/e2e/main_test.go#L78

Reply:
actually I thought it once, I choose environment because I feel it's easier to understand and extend in the future, but looks like we can have a description to let user know what's the flag for via option way. @antoninbas @tnqn any suggestion?

Reply:
Agreed, envvar is arbitrary, hard to remember and easy to have typo.

BTW, hard to understand what "NotDefault" means. Can we use some more meaningful feature words? For instances "slow", "disruptive". Then we can have an option to specify which words we want to skip:
```
--skip "slow, disruptive, windows"
```
And in the test case code, they can have an unified skip declartion to indicate under which situation the tests shouldn't run.
```
skipIfNotRequired("slow", "disruptive")
```